frontend/src/components/SearchLozenge/SearchLozenge.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './SearchLozenge.css';
+
+const SearchLozenge = (props) => {
+  let { searchTerm } = props;
+  searchTerm = searchTerm || '';
```

Comment:
I think @H12's suggestion was to use [`PropTypes`/`defaultProps`](https://reactjs.org/docs/typechecking-with-proptypes.html) to make sure this is set to a default value, instead of this `||` thing.